### ISSUE TITLE
Why is _.compact listed as an Array function?

### ISSUE THREAD

**Issue by captbaritone** [jashkenas/underscore#2622]
With it's current implementation, it works on Objects as well.

**Comment by amZotti** [comment]
Yeah, it seems this should be listed as a Collections function instead.

**Comment by captbaritone** [comment]
@amZotti Any interest in opening a PR?

**Comment by David2913** [comment]
I think this issue should be closed to avoid having so many opened issues, as wotkuni said, it was solved in #2655
